src/app/wallet/shared/address.service.ts

```diff
@@ -88,24 +99,29 @@ export class AddressService {
 
     return [offset, count];
   }
+*/
 
-  rpc_loadAddresses(response: Object): void {
-    this.deleteAddresses();
-    for (const k in response) {
-      if (response[k] !== undefined) { // lint
-        this.addAddress(response[k]);
+  private rpc_loadAddresses(json: Object): void {
+    let addresses: Address[] = [];
+    for (const k in json) {
```

Comment:
This should be called response, not json, and also this should do a forEach loop and not a for loop